### DISCUSSION TITLE
Have I bricked my new CC2652?

### DISCUSSION THREAD

**Discussion by Westcott1** [Koenkk/zigbee2mqtt#8608] [Devices]
I got one of these from AliExpress -
[https://www.aliexpress.com/item/1005002875838019.html](url)
Flashed it with CC1352P2_CC2652P_other_coordinator_20210708.hex .
The first reviewer noted which button to hold for boot mode, but didn't say which firmware to use.
Now it's not seen by z2m, and won't go into boot mode again.
Perhaps I should have used CC2652R_coordinator_20210708.hex

"Fortunately" it has a 5-pin cJTAG header.
Does anyone have experience of using this to re-flash?

**Comment by Westcott1** [comment]
UPDATE

Yes, I had bricked it.
Yes, I should have used the "R" firmware.
To recover, I got a CC-DEVPACK-DEBUG board from AliExpress, and a debug board from Electrolama.
Once I got the connections right, TI SmartRF 2 flashed it correctly first time.

**Reply by clint70** [reply]
any chance you can explain what you did... I think I did the same, bricked mine.

**Reply by Westcott1** [reply]
Hi Clint70,
In my case I bricked it by using the wrong firmware, 'P' rather than 'R'.
Once this has been done, it's not possible to directly reflash via USB.
Instead, if the stick has a JTAG header, you can use a combination of debug boards to flash directly onto the stick.
The pair of debug boards cost more than a new stick, but it was fun.
